src/components/Loader.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { CircularProgress } from '@material-ui/core';
+
+const Loader = () => {
+  return (
+    <div className="loading">
+      <CircularProgress />
```

Comment:
`CircularProgress` needs to have a `size` [prop](https://material-ui.com/api/circular-progress/#props). You included `size` on `Loader` but didn't pass it to `CircularProgress`.